src/qibolab/platform.py

```diff
@@ -2,22 +2,25 @@
 
 import math
 import re
-from dataclasses import asdict, dataclass, replace
+from dataclasses import asdict, dataclass, field, replace
 from pathlib import Path
 from typing import Dict, List, Optional
 
 import networkx as nx
 import yaml
 from qibo.config import log, raise_error
 
-from qibolab.channels import ChannelMap
 from qibolab.execution_parameters import ExecutionParameters
-from qibolab.instruments.abstract import Controller, Instrument
-from qibolab.native import NativeType, SingleQubitNatives, TwoQubitNatives
+from qibolab.instruments.abstract import Controller, Instrument, InstrumentId
+from qibolab.native import NativeType
 from qibolab.pulses import PulseSequence
 from qibolab.qubits import Qubit, QubitId, QubitPair, QubitPairId
 from qibolab.sweeper import Sweeper
 
+InstrumentMapType = Dict[InstrumentId, Instrument]
+QubitMapType = Dict[QubitId, Qubit]
+QubitPairMapType = Dict[QubitPairId, QubitPair]
```

Comment:
All `Type` suffixes are redundant, so they could be avoided (not that is a priority)